.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
we could do that, but we still want to publish the image only once after_success